src/entity/user/User.java

```diff
@@ -12,4 +14,5 @@ public interface User {
     void setEmail(String email);
     String getPhone();
     void setPhone(String num);
+    List addNotif(List<String> notif);
```

Comment:
this one can just be void i think, unless u want to return the List.

Reply:
so like void addNotifications(String notification);
atm i think we can have notifications to just be a list of strings, where the strings are the messages. Maybe in the future we can change it to notification objects, but i think it's okay for now